docs/services/passbolt.md

```diff
@@ -20,7 +20,7 @@ services:
     image: passbolt/passbolt
     restart: always
     ports:
-      - "[::1]:8000:80"
+      - "::1:8000:80"
```

Comment:
geht das?

Reply:
Ich glaube die Klammern werden benötig, da sonst yaml rumheult, da zwei doppelpunkte nebeneinander

Reply:
Das hat nichts mit YAML zu tun. In einer URI/URL müssen IPv6 Adressen in [] geschrieben werden, sodass der Port klar abgegrenzt ist.

Reply:
Und ::1 ist die IPv6 Adresse für den localhost (vgl. 127.0.0.1)